datalake/src/main/java/com/sequenceiq/datalake/flow/delete/handler/RdsDeletionHandler.java

```diff
@@ -54,7 +55,7 @@ protected void doAccept(HandlerEvent event) {
         Selectable response;
         try {
             sdxClusterRepository.findById(sdxId).ifPresent(sdxCluster -> {
-                if (sdxCluster.isCreateDatabase() && Strings.isNotEmpty(sdxCluster.getDatabaseCrn())) {
+                if (!SdxDatabaseAvailabilityType.NONE.equals(sdxCluster.getDatabaseAvailabilityType()) && Strings.isNotEmpty(sdxCluster.getDatabaseCrn())) {
```

Comment:
```
SdxDatabaseAvailabilityType.NONE.equals(sdxCluster.getDatabaseAvailabilityType())
```
is a re-occuring code, maybe in SDXcluster a new method cna be intoduce with this logic